R/dev.R

```diff
@@ -10,6 +10,11 @@
 #' @param credential_id The OpenStack application credential secret as a
 #'   character scalar
 #' @noRd
+#' @example 
+#' \dontrun{
+#'  metadata = CuratedAtlasQueryR::get_metadata() |> head(10) |> dplyr::collect()
+#'  update_database(metadata, "0.2.3", "rfypdlunhrfopdnkrs", "3q5lw3qntafptdfsrdh-wa4p8h")
```

Comment:
are these keys supposed to be public?

And what happens (could you please add to the example comment) when this command is launched?